src/html/HTMLWriter.jl

```diff
@@ -1405,20 +1405,76 @@ function edit_link(f, ctx, navnode)
-host_logo(remote::Remotes.GitHub) = host_logo_github
+function host_logo_github()
+    @tags svg path
+
+    # from SimpleIcons: https://github.com/simple-icons/simple-icons/blob/develop/icons/github.svg
+    # https://simpleicons.org/ License - https://github.com/simple-icons/simple-icons/blob/develop/LICENSE.md (CC0 1.0 Universal)
+    logo = svg[:viewBox => "0 0 24 24", :xlmns=>"http://www.w3.org/2000/svg"](
+        path[:d=>"M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"]
+    )
+
+    return (host = "GitHub", logo = logo)
+end
+
+function host_logo_bitbucket()
+    @tags svg path
+
+    # from SimpleIcons: https://github.com/simple-icons/simple-icons/blob/develop/icons/bitbucket.svg
+    # https://simpleicons.org/ License - https://github.com/simple-icons/simple-icons/blob/develop/LICENSE.md (CC0 1.0 Universal)
+    logo = svg[:viewBox => "0 0 24 24", :xlmns=>"http://www.w3.org/2000/svg"](
+        path[:d=>"M.778 1.211c-.424-.006-.772.334-.778.758 0 .045.002.09.01.134l3.263 19.811c.084.499.515.867 1.022.872H19.95c.382.004.708-.271.77-.646l3.27-20.03c.068-.418-.216-.813-.635-.881-.045-.008-.089-.011-.133-.01L.778 1.211zM14.52 15.528H9.522L8.17 8.464h7.561l-1.211 7.064z"]
+    )
+
+    return (host = "BitBucket", logo = logo)
+end
+
+function host_logo_gitlab()
+    @tags svg path
+
+    # from FontAwesome Brands: https://github.com/FortAwesome/Font-Awesome/blob/master/svgs/brands/gitlab.svg
+    # Font Awesome Free 5.15.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+    logo = svg[:viewBox => "0 0 512 512", :xlmns=>"http://www.w3.org/2000/svg"](
+        path[:d=>"M105.2 24.9c-3.1-8.9-15.7-8.9-18.9 0L29.8 199.7h132c-.1 0-56.6-174.8-56.6-174.8zM.9 287.7c-2.6 8 .3 16.9 7.1 22l247.9 184-226.2-294zm160.8-88l94.3 294 94.3-294zm349.4 88l-28.8-88-226.3 294 247.9-184c6.9-5.1 9.7-14 7.2-22zM425.7 24.9c-3.1-8.9-15.7-8.9-18.9 0l-56.6 174.8h132z"]
+    )
+
+    return (host = "GitLab", logo = logo)
+end
+
+function host_logo_azure()
+    @tags svg path
+
+    # from SimpleIcons: https://github.com/simple-icons/simple-icons/blob/develop/icons/azuredevops.svg
+    # https://simpleicons.org/ License - https://github.com/simple-icons/simple-icons/blob/develop/LICENSE.md (CC0 1.0 Universal)
```

Comment:
So.. apparently the Azure Devops logo has been removed from SimpleIcons on trademark grounds: https://github.com/simple-icons/simple-icons/pull/10019

This actually makes me a bit hesitant to, in general, include that actual icons here :cold_face:  Need to think about it a bit.

Reply:
I was not aware.

Personally I don't use Azure, I just wanted to help take care of some of the oldest PRs on this repo.

My view: if MS is hostile against users like this, then I would not bother providing them free PR by trying hard to show their branding: let's keep the generic icon for Azure links then.

Reply:
Haha I added this as a MSFT employee because we were using it! That project is long gone, but it would be nice to support azure still… can’t speak for the lawyers but I’d hope this is a reasonable use case…

Reply:
Well given that MS explicitly requested that simple icons remove that content, I'd personally not want to add this without explicit permission. In the end this is fringe enough to not warrant that headache, IMHO.

But in the end I'll leave it to Documenter maintainers to decide -- feel free to close this PR or merge it, I won't complain either way :-)

Reply:
Yea, let's remove the Azure one. It's their loss :man_shrugging:

Vendoring GitHub (https://github.com/logos) and Bitbucket (https://www.atlassian.com/legal/trademark#our-trademarks) shouldn't be a problem according to their trademark T&Cs. And FontAwesome looks good too.

Reply:
Also, we're currently using the Microsoft icon for Azure. It's not even clear if that's okay. So let's just fall back to the default icon (i.e. remove all Azure special casing).

Reply:
But is there any point to switching to SVG icons left then? Other than doing it to add Azure (and then doing it for all brands for consistency)?

(Don't get me wrong, I am happy to adjust this PR as indicated, I just wonder if that's really what you want?)

Reply:
Hmm. A fair point. It would allow us to still drop the dependency on the FontAwesome brands CSS file. But whether that's worth the complexity of maintaining the icons here, especially since we still keep the FontAwesome dep in general..

So.. I don't have super strong opinion about it to be honest. I'm also happy if we stick to the status quo, since it doesn't seem to cause any issues.

What's your preference here @fingolfin?

Reply:
Personally I'd prefer the status quo: it works right now so there is no risk of regression and I don't see how not having "FontAwesome brands CSS" helps us (as long as we use some parts of FontAwesome anyway...)

Reply:
Alright, then let's close these PRs -- I also lean towards the status quo. Sorry we didn't arrive at this conclusion earlier! At least this is a small case study in integrating icons and stuff directory into Documenter, so we can refer back to this if we need to.